services/auth-api/verifyUserMiddleware.js

```diff
@@ -1,38 +1,54 @@
 
-const { models, initDb } = require('../../infrastructure/db/');
+const { models } = require('../../infrastructure/db/');
+const logger = require('../../infrastructure/logging/logger');
 const config = require('./config');
 const jwt = require('jsonwebtoken');
 
+/**
+* Checks in the database if user with this email & password exists, if yes then continue to the next function
+* if no - then send 403K
+* @param {*} req express request
+* @param {*} res express response
+* @param {*} next passing to next middleware function
+*/
 const verifyLogin = async(req, res, next) => {
-
-    const user = await models.teammember.findOne({ where : {emailAddress : req.body.emailAddress} })
+    try {
+        const user = await models.teammember.findOne({ where : {emailAddress : req.body.emailAddress} });
+    } catch(err) {
+        throw Error(err);
+    }
     if(!user) {
-        res.status(404).send({error : "Invalid email address"});
+        res.status(403).send({error : "Invalid email address"});
+    }
+    let hashedPassword = req.body.password;
+    if(user.password == hashedPassword) {
+        req.body = {
+            id : user.id,
+            emailAddress : user.emailAddress
+        };
+        return next();
     }
     else {
-        let hashedPassword = req.body.password;
-        if(user.password == hashedPassword) {
-            req.body = {
-                id : user.id,
-                emailAddress : user.emailAddress
-            };
-            return next();
-        }
-        else {
-            res.status(404).send({ error : "Invalid email address or password" });
-        }
+        logger.warn('Invalid login' + req.body.emailAddress);
```

Comment:
Side note: note important at all but when you use ES6 you can use the ticking quotes to define a string. for example: 
logger.warn(`Invalid login ${req.body.emailAddress}`); 

by using the ticking quotes you can combine string with a variable without concatenation